css/amp.css

```diff
@@ -630,22 +621,6 @@ amp-list[resizable-children] > .i-amphtml-loading-container.amp-hidden {
   visibility: visible;
 }
 
-amp-list[load-more] > [load-more-button] > .i-amphtml-loader {
-  visibility: hidden;
```

Comment:
Danger! Moving these styles to `amp-list.css` will make the `[load-more]` child elements visible until `amp-list-0.1.js` loads. That's why AMP components like `amp-accordion` set style for elements that are initally-hidden in `amp.css`.

https://github.com/ampproject/amphtml/blob/56cc281d755df209aaeecdb305f3c003363b8c12/css/amp.css#L782-L785

Reply:
Yup you're right. I moved all the amp-list css below amp-accordion with a comment to that effect.